apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-applications/vo-applications.component.ts

```diff
@@ -82,7 +94,7 @@ export class VoApplicationsComponent implements OnInit {
         break;
       }
       case 'all': {
-        this.registrarManager.getApplicationsForVo(this.vo.id).subscribe(applications => {
+        this.registrarManager.getApplicationsForVo(this.vo.id, null, formatDate(this.startDate.value, 'yyyy-MM-dd','en-GB'), formatDate(this.endDate.value, 'yyyy-MM-dd','en-GB')).subscribe(applications => {
```

Comment:
on this place you miss the check to filter the applications for groups
you can rewrite part of code to 
`this.setData(null);
        break;`

Reply:
if you choose the option of rewriting, dont forget to do that in group-applications too

Reply:
Done.